pom.xml

```diff
@@ -519,26 +518,31 @@
                                 </goals>
                             </execution>
                         </executions>
+                        <configuration>
+                            <doclint>none</doclint>
+                            <excludePackageNames>com.dke.data.agrirouter.*</excludePackageNames>
+                        </configuration>
                     </plugin>
                 </plugins>
             </build>
         </profile>
         <profile>
             <id>jdk11</id>
             <activation>
-                <activeByDefault>false</activeByDefault>
+                <activeByDefault>true</activeByDefault>
                 <jdk>11</jdk>
             </activation>
             <properties>
-                <jdk.dependent.jar.classifier>jdk11</jdk.dependent.jar.classifier>
+                <maven.compiler.source>11</maven.compiler.source>
+                <maven.compiler.target>11</maven.compiler.target>
+                <agrirouter-api-protobuf-definitions.version>1.0.0-jdk11</agrirouter-api-protobuf-definitions.version>
             </properties>
             <dependencyManagement>
                 <dependencies>
                     <dependency>
                         <groupId>com.agrirouter.proto</groupId>
-                        <artifactId>definitions</artifactId>
-                        <version>${definitions.version}</version>
-                        <classifier>jdk11</classifier>
+                        <artifactId>agrirouter-api-protobuf-definitions</artifactId>
+                        <version>${agrirouter-api-protobuf-definitions.version}</version>
```

Comment:
Is this variable even available? It has been deleted from former line 71. Do I miss a redefinition somewhere?

Reply:
Line 538, the property is defined for each build profile.